doc/ui-guidelines.md

```diff
@@ -1,5 +1,18 @@
 # UI components coding guidelines
 
+## Content
+- [Global State and Subscriptions](#global-state-and-subscriptions)
+  - [Regular Atoms](#regular-atoms)
+  - [Effects](#effects)
```

Comment:
atoms and effects are not part for global state and subs, they should be on the same level